env.example

```diff
@@ -22,3 +22,7 @@ export DEPLOY_OS=centos7
 # customise this section
 #export IMAGE_RESOURCEGROUP=images
 #export IMAGE_RESOURCENAME=$(az image list -g $IMAGE_RESOURCEGROUP -o json --query "[?starts_with(name, '${DEPLOY_OS:-rhel7}-$DEPLOY_VERSION') && tags.valid=='true'].name | sort(@) | [-1]" | tr -d '"')
+
+# If set to 'true', the sync process will run locally. Useful for
+# development purposes only.
+export RUN_SYNC_LOCAL=true
```

Comment:
should that be exported by default?

Reply:
I exported by default to continue run `sync` as it runs now. I don't mind commenting out.